addon/components/hyper-table-v2/index.hbs

```diff
@@ -1,11 +1,23 @@
 <div class="hypertable-container" ...attributes>
-  <div class="hypertable__upper-header">
-    <div class="left-side">
-      Search {{!--to do implemented--}}
-    </div>
-    <div class="right-side">
-      <HyperTableV2::ManageColumns @handler={{@handler}} />
-    </div>
+  <div class="hypertable__upper-header fx-row fx-malign-space-between">
+    {{#if this.features.searchable}}
+      {{#if (has-block "search")}}
+        {{yield to="search"}}
```

Comment:
`{{yield @handler to="search"}}` so we can delay some features to the parent

Reply:
Resolving as double checked with you ; handler is already defined by the parent, no need to yield it back.